process_manager/views/partials.py

```diff
@@ -40,6 +40,16 @@ def process_table(request: HttpRequest) -> HttpResponse:
                 "exit_code": process_instance.return_code,
             }
         )
+    # Filter table data based on search parameter
+    if search := request.GET.get("search", ""):
+        table_data = [
+            row
+            for row in table_data
+            if any(
+                search in row[k]
+                for k in ["uuid", "name", "user", "session", "status_code"]
```

Comment:
The problem with including the exit code is that it is just, typically, a 1-digit number, so filtering will pick every row with that number, being that in the uuid, the name, or anywhere. 

We could easily add modifiers to make the search column specific. By default, it searchers in all columns, but if you write `column_name:search_term`, it only searches in that column. For example, `exit_code:4` will filter rows based only on what it can be found in `exit_code` column.

Reply:
Fair point. Let's leave as is then and see what feedback we get.

Reply:
Check new version.